src/editors/containers/VideoEditor/components/VideoSettingsModal/components/LicenseWidget/LicenseDetails.jsx

```diff
@@ -55,7 +57,7 @@ const LicenseDetails = ({
                     <CheckboxControl
                       disabled
                       checked
-                      aria-label="Checkbox"
+                      aria-label={intl.formatMessage(messages.attributionCheckboxAriaLabel)}
```

Comment:
```suggestion
```

I think you can actually just remove the `aria-label` altogether, because this control is inside a `Form.Group` which provides a `Form.Label` for this control.

I confirmed in Chrome that if we just remove `aria-label`, it gets a _more accurate_ label ("Attribution") and still has role: checkbox, so I don't think there's any need for a generic label to tell the user this is a checkbox. I think that's actually making it harder to use for non-visual users.

![Labelled Checkbox](https://github.com/user-attachments/assets/64f28dad-2cca-470b-b387-b01801caea48)